src/pages/LandingPage/LandingPage.jsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react'; // <-- useState is a react hook and imported here, you must always import react hooks
 import scrollToElement from '../../shared/utility/scrollToElement';
 import PortfolioCard from './PortfolioCard';
 import ContactForm from './ContactForm';
 import Render3DShape from './Render3DShape';
 
+const POSSIBLE_SHAPES = [
+  'torus',
+  'sphere',
+  'dodecahedron',
+  'icosahedron',
+  'octahedron',
+  'tetrahedron',
+  'torusknot',
+  'sahilsknot'
+];
+
+const INITIAL_SHAPE_INDEX = 0;
+const INITIAL_SHAPE = POSSIBLE_SHAPES[INITIAL_SHAPE_INDEX];
```

Comment:
This is just fixing my own work basically: Its a good convention/pattern to write constants in ALL_CAPS, this signifies to other developers that this is a "hardcoded" value and not something that changes during runtime. 

I also added a INITIAL_SHAPE_INDEX & INITIAL_SHAPE constant so we can programmatically compare the intial shape against the first random shape when running handleClick the first time

Reply:
Yes, it shows that it is more important and shouldn't be changed.